src/test/mocks/store.ts

```diff
@@ -112,4 +114,8 @@ export const mockStore: ReduxState = {
     byValidator: {},
     validatorIds: [],
   },
+  foxEth: {
+    farmOpportunities: [],
+    lpOpportunity: {} as EarnOpportunityType,
```

Comment:
Might or might not be early/over engineering, but do we want to have these two under the `opportunities` umbrella and give them a `type`?